spacepy/pybats/rim.py

```diff
@@ -241,7 +241,48 @@ def readascii(self):
         # Some extra grid info:
         self.dlon = self['n_psi'  ][0,3]-self['n_psi'  ][0,2]
         self.dlat = self['n_theta'][3,0]-self['n_theta'][2,0]
-        
+
+    def calc_j(self):
```

Comment:
Should this (and others) use calc_wrapper? Or are they fast enough not to matter?

Reply:
No. `calc_wrapper` is for multi-frame files (*.outs); files are not organized that way in RIM.

Reply:
Cool, thanks.